spring-dubbo-practice/dubbo-provider/src/main/java/com/dante/spring/dubbo/impl/FirstDubboServiceImpl.java

```diff
@@ -11,6 +11,7 @@ public String sayHello() {
         int key = 1/0;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make sure this expression can't be zero before doing this division. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.99.100:9000/coding_rules#rule_key=squid%3AS3518)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "key". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.99.100:9000/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "key" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.99.100:9000/coding_rules#rule_key=squid%3AS1481)